crates/holochain_cascade/src/lib.rs

```diff
@@ -644,8 +645,8 @@ where
             // We got a result so can short circuit.
             if results.is_some() {
                 return Ok(results);
-            // We didn't get a result so if we are either authoring
-            // or the authority there's nothing left to do.
+                // We didn't get a result so if we are either authoring
+                // or the authority there's nothing left to do.
```

Comment:
too much indentation?

Reply:
This was the formatter and it's technically right because the content of the `if` block should be indented. The comments would probably be better off inside the block below that it refers to. 

I've put this back to how it was for now